src/patterns/filtering/multiple-categories.stories.ts

```diff
@@ -0,0 +1,200 @@
+        { color: "Green", checked: false },
+        { color: "White", checked: false }
+    ]
+
+    checkboxList = [
+        { value: "Fruit", checked: false},
+        { value: "Vegetable", checked: false},
+        { value: "Meat", checked: false}
+    ]
+
+    onCheckboxChange(event) {
+        if (event.checked) {
+            this.checkBoxFilters.add(event.source.value);
+        } else {
+            this.checkBoxFilters.delete(event.source.value);
+        }
+        this.applyFilters();
+    }
+
+    onRadioChange(event) {
+        this.radioFilter = event.value;
+        this.applyFilters();
+    }
+
+    resetFilters() {
+        this.manyRadios = this.manyRadios.map(obj => {
+            return { color: obj.color, checked: false }
+        });
+
+        this.checkboxList = this.checkboxList.map(obj => {
+            return { value: obj.value, checked: false }
+        });
+
+        this.radioFilter = null;
+        this.checkBoxFilters = new Set();
+
+        this.applyFilters();
+    }
+
+    applyFilters() {
+        this.model.data = [];
+        this.model.data.pop();
+
+        this.dataset.forEach(data => {
+            if((this.applyCheckboxFilters(data.type) || !this.checkBoxFilters.size) && (data.color === this.radioFilter || !this.radioFilter) ) {
+                this.model.data.push(
+                    [
+                        new TableItem({ data: data.name }),
+                        new TableItem({ data: "Lorem ipsum dolor sit amet, consectetur adipiscing elit." })
+                    ]
+                )
+            }
+        });
+    }
+
+    applyCheckboxFilters(types: Array<String>): Boolean {
+        let shouldInclude = true;
+        this.checkBoxFilters.forEach(filter => {
+            if (!types.includes(filter)) {
+                shouldInclude = false;
+            }
+        });
+        return shouldInclude;
+    }
+
+    ngOnInit() {
+        this.model.header = [
+            new TableHeaderItem({
+                data: "Name"
+            }),
+            new TableHeaderItem({
+                data: "Description"
+            })
+        ]
+        
+        this.model.data = this.dataset.map(datapoint => 
+            [
+                new TableItem({ data: datapoint.name}),
+                new TableItem({ data: "Lorem ipsum dolor sit amet, consectetur adipiscing elit." })
+            ]
+        );
+    }
+}
+
+storiesOf("Patterns|Filtering", module)
+	.addDecorator(
+		moduleMetadata({
+			declarations: [ SampleMultiCategories ],
+			imports: [
+                TableModule,
+                DropdownModule,
+                GridModule,
+                StructuredListModule,
+                RadioModule,
+                CheckboxModule,
+                ButtonModule
+			]
+		})
+	)
+	.addDecorator(withKnobs)
+	.add("Multiple Categories", () => ({
+        template: `
+            <app-sample-multi-categories></app-sample-multi-categories>
+            `
+	}))
```

Comment:
Needs a newline and a semicolon
```suggestion
	}));

```